package.json

```diff
@@ -198,7 +198,8 @@
     "lint-fmt": "prettier --check .",
     "test": "node --test out/test/",
     "ui-test-scarb": "extest setup-and-run './out/ui-test/scarb/*.js' --code_version max --code_settings ui-test/settings.json --extensions_dir .test-extensions",
-    "ui-test": "extest setup-and-run './out/ui-test/generic/*.js' --code_version max --code_settings ui-test/settings.json --extensions_dir .test-extensions"
+    "ui-test": "extest setup-and-run './out/ui-test/generic/*.js' --code_version max --code_settings ui-test/settings.json --extensions_dir .test-extensions",
+    "ui-test-syntax-tree": "extest setup-and-run './out/ui-test/syntaxTree/*.js' --code_version max --code_settings ui-test/settings.json --extensions_dir .test-extensions"
```

Comment:
maybe replace this litany od specific scripts with a simple bash script that runs this incantation for each `./out/ui-test/*` directory in a for loop?